sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/throughputControl/ThroughputControlTests.java

```diff
@@ -208,6 +209,25 @@ public void throughputLocalControlWithThroughputQuery() {
         }
     }
 
+    @Test(groups = {"emulator"}, dataProvider = "operationTypeProvider", timeOut = TIMEOUT)
+    public void throughputLocalControlPriorityLevel(OperationType operationType) {
+        ThroughputControlGroupConfig groupConfig =
```

Comment:
I think for the group config build, we can use the existing builder pattern. For example:

```
   ThroughputControlGroupConfig groupConfig =
            new ThroughputControlGroupConfigBuilder()
                .priorityLevel(PriorityLevel.Low)
                .build();
```

Reply:
Priority level can be configured for both local and global throughput control groups. If customers are not using throughput control groups, but want to use the priority based throttling feature, then local is preferred since it doesn't have the overhead of using a meta store.

Reply:
thanks Bala, I will resolve this comment for now, as I think the questions rises around how we want to integrate with client side throughput control with server side throughput control